test/erlang_vm_schema_tests.erl

```diff
@@ -88,8 +88,8 @@ override_schema_test() ->
     %% These settings are version dependent, so we won't even test them here
     %% because we don't know what version you're running, so we'll cover it
     %% in two tests below
-    case erlang:system_info(otp_release) of
-        [$R, $1, N|_] when N >= $6 ->
+    case cuttlefish:otp("R16", erlang:system_info(otp_release)) of
```

Comment:
Using the `cuttlefish:otp/2` function here is OK, provided the unit test for this function has asserted that the function is working correctly. As an alternative, the conditional code blocks could be swapped around and guarded by `[$R, $1, N|_] when N =< $5`